graphium/data/datamodule.py

```diff
@@ -790,8 +788,7 @@ class MultitaskFromSmilesDataModule(BaseDataModule, IPUDataModuleModifier):
     def __init__(
         self,
         task_specific_args: Union[Dict[str, DatasetProcessingParams], Dict[str, Any]],
-        processed_graph_data_path: Optional[Union[str, os.PathLike]] = None,
-        dataloading_from: str = "ram",
```

Comment:
I can probably add something for this interface.  Some of the other interfaces, like `MultitaskDataset` are changed too drastically for a deprecation warning to suffice, but I don't know which interfaces are expected to be directly used by users, instead of users using only the command line and yaml files, so maybe some of those other interfaces aren't a concern.

Reply:
I've added them back with deprecation warnings when used, and removed them from all yaml files.

Reply:
I hate to contradict myself, but if we release a version 2.0, with the expectations that things break, can we just remove this options without the deprecation warning? I would keep a note in the docstring.